partner_multi_company/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Simone Rubino - PyTech
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        companies = super().create(vals_list)
+        for vals, company in zip(vals_list, companies):
+            if "user_ids" in vals:
+                for user in company.user_ids:
+                    user.partner_id.company_ids += company
```

Comment:
No need to over-complicate:

```suggestion
        for company in companies
            for user in company.user_ids:
                    user.partner_id.company_ids += company
```

Reply:
:+1: I noticed this in the user's override but not in the company one, thanks! Please check